syntax = "proto2";

message AnnotatedTree {
  required string name = 1;
  repeated int32 nodes = 2;
  repeated int32 depth = 3;
  repeated int32 from_node = 4;
  repeated int32 assignment = 5;
  repeated int32 assign_attr = 6;
  repeated int32 position = 7;
}
